tests/ui/perfherder/alerts_test.jsx

```diff
@@ -202,6 +203,43 @@ test('toggle buttons should filter alert summary and alerts by selected filter',
   expect(alert2).not.toBeInTheDocument();
 });
 
+test('alert filtering ignores repository and options, but filters by platform', async () => {
+  const { getByPlaceholderText, getByTestId } = alertsView();
```

Comment:
We need 2 extra & separate tests for the warning notification: one for checking `options`, one for repositories.